mail-manager/src/test/java/inc/softserve/health/MailManagerHealthCheckTest.java

```diff
@@ -0,0 +1,82 @@
+package inc.softserve.health;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.mongodb.CommandResult;
+import com.mongodb.DB;
+import inc.softserve.common.EnvelopeTools;
+import inc.softserve.dao.MongoManaged;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class MailManagerHealthCheckTest extends Assert {
+
+    private MailManagerHealthCheck healthCheck;
+    private EnvelopeTools envTools;
+    private MongoManaged mongo;
+    private final String HEALTHY_ESP_MESSAGE = "{\"deadlocks\":{\"healthy\":true},\"health\":{\"healthy\":true}}";
+
+    @BeforeClass
+    public void setUpDataSource() throws Exception {
+        envTools = mock(EnvelopeTools.class);
+        mongo = mock(MongoManaged.class);
+        this.healthCheck = new MailManagerHealthCheck("","", envTools, mongo);
+    }
+
+    @Test
+    public void testCheckHealthy() {
+        DB dbMock = mock(DB.class);
+        CommandResult statsResultMock = mock(CommandResult.class);
+        when(mongo.getDb()).thenReturn(dbMock);
```

Comment:
That statement with the first one looks to be good candidates to go to `@BeforeTest` method